src/unix/apple/macos/component/arm.rs

```diff
@@ -101,26 +96,27 @@ impl ComponentsInner {
             let count = CFArrayGetCount(services.inner());
 
             for i in 0..count {
-                let service = match CFReleaser::new(
-                    CFArrayGetValueAtIndex(services.inner(), i) as *const _
-                ) {
-                    Some(s) => s,
-                    None => continue,
-                };
+                let service = CFArrayGetValueAtIndex(services.inner(), i);
```

Comment:
Please add a comment mentioning that `service` should never be freed (and add link to the issue).